package.json

```diff
@@ -63,11 +63,14 @@
   "devDependencies": {
     "@typescript-eslint/eslint-plugin": "^4.14.2",
     "@typescript-eslint/parser": "^4.14.2",
-    "eslint": "^7.19.0",
+    "eslint": "^7.32.0",
     "eslint-config-airbnb": "^18.2.1",
+    "eslint-config-prettier": "^8.3.0",
     "eslint-plugin-import": "^2.22.1",
     "eslint-plugin-jsx-a11y": "^6.4.1",
+    "eslint-plugin-prettier": "^4.0.0",
     "eslint-plugin-react": "^7.22.0",
-    "eslint-plugin-react-hooks": "^4.2.0"
+    "eslint-plugin-react-hooks": "^4.2.0",
+    "prettier": "^2.4.1"
   }
 }
```

Comment:
Not seeing a script for prettier

You could either add it to the "lint" script or make a "format" script, but right now you are relying on a custom CLI command that you provided in the PR but no where else, or the editor, but it should be possible to run prettier with a script so everyone has the same result

Goal of all this setup is that everyone has the same result, machines, ides or humans